docs/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 All notable changes to this project will be documented in this file.
 We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
+## v4.0.6.1 - 2022-08-11 - [PR #654](https://github.com/NOAA-OWP/inundation-mapping/pull/654)
+
+`inundate_nation.py` A change to switch the inundate nation function away from refrences to `inundate.py`, and rather use `inundate_gms.py` and `mosaic_inundation.py`
+## Changes
+
+- `inundate_gms`:  Changed line 214 to use `mask_type = 'filter'`
+
+<br/><br/>
+
+=======
```

Comment:
These = are special characters and cause issues. Let's take them out. Also, let's make sure that the changelog Changes and Additions sections match your PR.